anesthetic/plot.py

```diff
@@ -281,6 +292,13 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
 
     ans = ax.plot(x[i], p[i], color=color, *args, **kwargs)
     ax.set_xlim(*check_bounds(x[i], xmin, xmax), auto=True)
+
+    if fill is True:
+        c = iso_probability_contours_from_samples(p[i], contours=levels)
+        for j in range(len(c)-1):
+            ax.fill_between(
+                x[i], p[i], where=(p[i] >= c[j]), color=cmap(levels[j]))
```

Comment:
Following the rest of the anesthetic conventions, this would probably be better formatted as
```python
            ax.fill_between(x[i], p[i], where=p[i] >= c[j], 
                            color=cmap(levels[j]))


```